libpod/runtime_ctr.go

```diff
@@ -905,3 +906,67 @@ func (r *Runtime) PruneContainers(filterFuncs []ContainerFilter) (map[string]int
 	}
 	return prunedContainers, pruneErrors, nil
 }
+
+// StorageContainers returns a list of containers from containers/storage that
+// are not currently known to Podman.  The list of containers passed in are all
+// of the containers known to Podman from the GetContainers() function.
+func (r *Runtime) StorageContainers() ([]*Container, error) {
```

Comment:
We can't make this return Libpod containers - having a bunch of semi-functional things wandering around calling themselves a libpod.Container is asking for trouble (they don't have valid locks, for example, we'll panic the moment someone tries to call a real Libpod function on one of them). I think we need a separate API for this, that returns something like []*StorageContainer - we can translate those into the PS container struct in pkg/domain, so the code in PS will remain unchanged.